spec/dummy/db/migrate/20181227144320_create_multi_conversations.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMultiConversations < ActiveRecord::Migration[5.2]
```

Comment:
I'm not sure if it's a pro or cons but the solution on this PR allows only 1 "type" of conversations at a time (multichat or not) and the `admin_user_id: nil` solution supports both at the same time.

The `admin_user_id` solution seems more straight forward from a user point of view, if conversation has an admin_user id it belongs to it, if it's `nil` it belongs to every admin. It will also simplify tests. Thinking on the code I guess `admin_user_id` solution can make things harder since we have to support both things at the same time. For example when an admin creates a conversation by clicking the `Send message` button in active admin, should the gem create a conversation just for that admin or for every admin?

In a nutshell, I think we should go with the solution that makes more sense from a product point of view. Do we want to support both "types" of chats or not?
IMO supporting both "types" of conversations can be positive since we leave the choice up to the user what they want to do (and they won't even need to set a new flag when installing the gem)

Reply:
Agree. I would go with the `admin_user_id` nil solution.